Doc/library/pathlib.rst

```diff
@@ -222,6 +222,8 @@ The slash operator helps create child paths, similarly to :func:`os.path.join`::
    >>> q = PurePath('bin')
    >>> '/usr' / q
    PurePosixPath('/usr/bin')
+   >>> p / '/another' / 'absolute' / 'path'
+   PurePosixPath('/another/absolute/path')
```

Comment:
```suggestion
   >>> p / '/usr' / 'lib64'
   PurePosixPath('/usr/lib64')
```

It's fairly easy to miss the leading `/`. Using a well-known top-level path like '/usr/lib64' makes it more obvious that we've passed an absolute path.

Reply:
I agree with the problem, which is why I tried to make the path self-descriptive.
The reason I didn't use `/usr/lib` is because it looks a lot like the previous examples, so it's easy for users to miss what's happening.

I think just `p / "/an_absolute_path"` would be clearer. I'll make this change, let me know if you still think the example would be confusing for readers!

Reply:
Yep I think you're right. For some reason I naturally assume that the string literals are 'dummy values' that can be ignored when trying to understand the concept. Your new version is perfect IMO.